sentences/fr/valve_HassTurnOn.yaml

```diff
@@ -0,0 +1,11 @@
+language: fr
+intents:
+  HassTurnOn:
+    data:
+      - sentences:
+          - "ouvre [<le>]{name}"
```

Comment:
Same comment as for HassSetPosition : Verbs should be expansion rules and support infinitive forms